app/views/search/search.html.erb

```diff
@@ -3,81 +3,74 @@
     t('.search_results') %><% end %>
 
 <form action="/search" method="GET">
-  <main role="main" id="content">
-  <h1 class="heading-large"> <%= t('.search_results_heading') %></h1>
-  <div class="grid-row">
-    <div class="column-two-thirds dgu-search-box">
-      <label for="q" class="visuallyhidden"> <%= t('.accessibility.search_box_label') %></label>
-      <input id="q"
-             name="q"
-             type="text"
-             class="form-control dgu-search-box__input"
-             value="<%= @query %>"/><button type="submit"
-                                            class="dgu-search-box__button">
-               <%= t('.accessibility.search_button_label') %>
-             </button>
-    </div>
-  </div>
-
-  <div class="grid-row dgu-filters">
-    <div class="column-one-third">
-      <%= render 'filters' %>
-    </div>
+  <main role="main" id="main-content" class="govuk-main-wrapper">
+    <div class="govuk-width-container">
+      <h1 class="govuk-heading-l"> <%= t('.search_results_heading') %></h1>
```

Comment:
This could be a [heading component](https://components.publishing.service.gov.uk/component-guide/heading):
```suggestion
      <%= render "govuk_publishing_components/components/heading", {
        text: t('.search_results_heading'),
        heading_level: 1,
        font_size: "l",
      } %>
```